### DISCUSSION TITLE
Do changes on files happen mutually (webDAV) ?

### DISCUSSION THREAD

**Discussion by DeepBlueBubble** [remotely-save/remotely-save#200] [Q&A] [answered]
Hi, I am pretty much a technical idiot here, so please, please bear with me. Thanks

I use the plugin as secondary method for backup, just in case if my primary one fail or something. So, my intention is that, the file can be uploaded to the cloud and will not change the local one, in any situation. In other word, I wish the **sync only happen in one way**.
So my question is, would my local file be changed, if I use this plugin? Let's say, I changed something on the cloud, and then it changes back to my local file (and it is not something I want to see).

I use webDAV of NextCloud.

**Comment by guzhouqiusheng** [comment, marked as answer]
You maybe just want to have a copy in your cloud, but actually the sync will transfer notes from the newer modified time to the older one. If you can keep not changing your cloud files, I think every time you can successfully copy.
